docs/source/howto/draft.md

```diff
@@ -26,19 +26,19 @@ cd zeromq-${ZMQ_VERSION}
 make -j${CPU_COUNT} && make install
 ```
 
-And to install pyzmq with draft support:
+And then build pyzmq with draft support:
 
 ```bash
 export ZMQ_PREFIX=${PREFIX}
 export ZMQ_DRAFT_API=1
-pip install -v --no-binary pyzmq --pre pyzmq
+pip install -v --no-binary pyzmq pyzmq
```

Comment:
I am not a `pip` expert, but shouldn't this line be either just

`pip install -v --no-binary pyzmq`

or 

`pip install -v --no-binary :pyzmq: pyzmq`

or

`pip install -v --no-binary :all: pyzmq`

Reply:
Thanks for checking! The line is correct. `--no-binary` takes an argument for which package(s) to install from source, and then remaining package names are what to install. `:all:` is a special string that's not a valid package name, meaning install no binaries at all, which you don't actually want because that prevents e.g. installing wheels of build dependencies like `cython`. Since pyzmq is the only package you want to install from source, you want `--no-binary pyzmq`.

Reply:
Make sense. Thank you for the explanation.